cli.js

```diff
@@ -8,10 +8,11 @@ yaxys.init()
     const argv = minimist(process.argv.slice(2));
     const commandsHash = {
       async init_db() {
-        yaxys.logger.error("Not implemented yet");
+        await ModelService.createTableForAllModels();
+        yaxys.logger.info("done");
       },
       get_sql() {
-        yaxys.logger.error("Not implemented yet");
+        ModelService.getSQLForAllModels();
```

Comment:
That is not right. Function `getSQLForAllModels` should just return SQL string as a result. The console output should be done here. Instead, you do the output inside of `getSQLForAllModels` function, which doesn't correspond with its name.